gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/api/use_case/UpdateFederatedApiUseCase.java

```diff
@@ -67,7 +71,7 @@ public Output execute(Input input) {
 
         var validatedUpdatedApi = validateFederatedApiDomainService.validateAndSanitizeForUpdate(updateApi, currentApi, primaryOwnerEntity);
```

Comment:
I wonder if this method should contain the category validation since we need categories to be sanitized every time?

Reply:
No, because inside this method we change only updateApi which at this point does not have categories from updateApi.

It will happen later down in this method.

So there is no point checking on categories in updateApi which are empty for now.